internal/app/machined/pkg/system/services/etcd.go

```diff
@@ -184,6 +184,11 @@ func (e *Etcd) DependsOn(runtime.Runtime) []string {
 	return []string{"cri"}
 }
 
+// Volumes implements the Service interface.
+func (e *Etcd) Volumes() []string {
+	return nil
```

Comment:
doesn't etcd need ephemeral?

Reply:
it needs `/var/lib/etcd`, but I don't have this implemented yet :) so I will add this with directory volumes